### PR TITLE
[MST] Add MFA procedure, and content about auth tokens

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Add MFA procedure from Kbase.

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

**Review by alexthinath — COMMENTED** [review]
I think we need to double check this functionality, I'm not sure this works how its described and was just a copy-paste from Aiven's docs a long time ago, before my time :)

**Comment by Loquacity** [comment]
> I think we need to double check this functionality, I'm not sure this works how its described and was just a copy-paste from Aiven's docs a long time ago, before my time :)

I can tell you that from the UI perspective, it works as described because I tested it. As to the backend stuff, though, if you want to double-check, then please do 👍

File: mst/user-management.md [hunk 1]
```diff
@@ -85,5 +90,88 @@ Your service must be running before you can manage users.
 
 </procedure>
 
+## Multi-factor user authentication
+
+You can use multi-factor authentication (MFA) to log in to your Managed Service
+for TimescaleDB account. This requires an authentication code, provided by the
+Google Authenticator app on your mobile device, to be able to log in to your
+Managed Service for TimescaleDB account.
```
Comment (charislam) [inline mst/user-management.md:98]:
```suggestion
Google Authenticator app on your mobile device.
```

Non-blocking suggestion, found this a bit repetitive

File: mst/user-management.md [hunk 2]
```diff
@@ -85,5 +90,88 @@ Your service must be running before you can manage users.
...
+
+You can see which authentication method is in use by each member of your Managed
+Service for TimescaleDB project. From the dashboard, navigate to the `Members`
+section. Each member is listed in the table with an authentication method of
+either `Password`, or `Two-Factor`.
```
Comment (charislam) [inline mst/user-management.md:103]:
```suggestion
either `Password` or `Two-Factor`.
```

File: mst/user-management.md [hunk 3]
```diff
@@ -85,5 +90,88 @@ Your service must be running before you can manage users.
...
+1.  On your mobile device, scan the QR code provided by Managed Service for
+    TimescaleDB.
+1.  In your Managed Service for TimescaleDB dashboard, enter the confirmation
+    code provided by the Google Authenticator app, and click
+    `Enable Two-Factor Auth`.
+
+</procedure>
+
+<highlight type="warning">
+If you have lost access to the mobile device uses for your multi-factor
```
Comment (charislam) [inline mst/user-management.md:129]:
```suggestion
If you have lose access to the mobile device you use for multi-factor
```

File: mst/user-management.md [hunk 4]
```diff
@@ -85,5 +90,88 @@ Your service must be running before you can manage users.
...
+regained access to your account, reconfigure multi-factor authentication.
+</highlight>
+
+## User authentication tokens
+
+Every time a registered user logs in, Managed Service for TimescaleDB creates a
+new authentication token. This occurs for login events using the portal, and
+using the API. By default, authentication tokens expire after 30 days, but the
+expiry date is adjusted every time the token is used. This means that tokens can
+be used indefinitely, if the user logs in at least every thirty days.
```
Comment (charislam) [inline mst/user-management.md:142]: Not terribly bothered by whether we spell out `thirty` or write `30`, but we should make it consistent within this paragraph.

File: mst/user-management.md [hunk 5]
```diff
@@ -85,5 +90,88 @@ Your service must be running before you can manage users.
...
+Additionally, you can click `Generate token` to create a new token. When you
+generate a token on this page, you can provide a description, maximum age, and
+an extension policy. Generating authentication tokens in this way allows you to
+use them with monitoring applications that make automatic API calls to Managed
+Service for TimescaleDB.
+
+<highlight type="important">
+There is a limit to how many valid authentication tokens are allowed per user.
+This limit is different for tokens that are created as a result of a sign in
+operation, and for tokens created explicitly. For automatically-created tokens,
```
Comment (charislam) [inline mst/user-management.md:165]:
```suggestion
operation, and for tokens created explicitly. For automatically created tokens,
```

File: mst/user-management.md [hunk 6]
```diff
@@ -85,5 +90,88 @@ Your service must be running before you can manage users.
...
+the system automatically deletes the oldest tokens as new ones are creates. For
```
Comment (charislam) [inline mst/user-management.md:166]:
```suggestion
the system automatically deletes the oldest tokens as new ones are created. For
```

File: mst/user-management.md [hunk 7]
```diff
@@ -85,5 +90,88 @@ Your service must be running before you can manage users.
...
+explicitly-created tokens, older tokens are not deleted unless they expire or
```
Comment (charislam) [inline mst/user-management.md:167]:
```suggestion
explicitly created tokens, older tokens are not deleted unless they expire or
```

File: mst/user-management.md [hunk 8]
```diff
@@ -85,5 +90,88 @@ Your service must be running before you can manage users.
...
+are manually revoked. This can result in explicitly-created tokens that stop
```
Comment (charislam) [inline mst/user-management.md:168]:
```suggestion
are manually revoked. This can result in explicitly created tokens that stop
```

**Review by charislam — APPROVED** [review]

File: mst/user-management.md (hunk 4 above)
Comment (Loquacity) [reply mst/user-management.md:142]: oof, good spot!

```suggestion
be used indefinitely, if the user logs in at least every 30 days.
```

**Review by alexthinath — APPROVED** [review]

**Pull request merged**